packages/cli/src/commands/import.ts

```diff
@@ -0,0 +1,60 @@
+import { Command, flags } from '@oclif/command';
+import { promises as fs } from 'fs';
+import { Messages } from '../constants/messages.constants';
+import { parseDataFiles } from '../libs/data';
+
+export default class Import extends Command {
+  public static description = 'Import a mock API';
```

Comment:
I would be more explicit about the source format:
```suggestion
  public static description = 'Import a OpenAPI v3 specification file (YAML or JSON)';
```

Reply:
I will also add that it is possible to import a Swagger v2 file.
What do you think?

Reply:
Yes true, import supports both v2 and v3